tests/cloudpickle_test.py

```diff
@@ -2352,6 +2371,71 @@ def test_lookup_module_and_qualname_stdlib_typevar():
     assert name == 'AnyStr'
 
 
+def test_lookup_module_and_qualname_external_module_remove():
+    import _cloudpickle_testpkg
+    T = _cloudpickle_testpkg.T
+
+    register_pickle_by_value(_cloudpickle_testpkg)
+    unregister_pickle_by_value(_cloudpickle_testpkg)
+    module_and_name = _lookup_module_and_qualname(T, name=T.__name__)
+    assert module_and_name is not None
+
+
+def test_lookup_module_and_qualname_external_module_remove_name():
+    import _cloudpickle_testpkg
+    T = _cloudpickle_testpkg.T
+
+    register_pickle_by_value("_cloudpickle_testpkg")
+    unregister_pickle_by_value("_cloudpickle_testpkg")
+    module_and_name = _lookup_module_and_qualname(T, name=T.__name__)
+    assert module_and_name is not None
+
+
+def test_lookup_module_and_qualname_external_module():
```

Comment:
(nitpick) As for the other test, maybe we can align the naming with the convention discussed above, and merge with the test below.